ci_build.sh

```diff
@@ -29,13 +30,12 @@ if [ $BUILD_TYPE == "default" ]; then
         CONFIG_OPTS+=("--enable-drafts=yes")
     fi
 
-    #   Build and check this project
-    (
-        ./autogen.sh &&
-        ./configure "${CONFIG_OPTS[@]}" &&
-        export DISTCHECK_CONFIGURE_FLAGS="${CONFIG_OPTS[@]}" &&
-        make VERBOSE=1 distcheck
-    ) || exit 1
+    # Build and check this project
+    ./autogen.sh
+    ./configure "${CONFIG_OPTS[@]}"
+    make -j5
+    make check
+    make install
```

Comment:
I think so, otherwise we won't notice if make dist is broken until we try to do a release, which is not nice :-)